demo-java/app/src/main/java/com/example/placesdemo/programmatic_predictions/ProgrammaticAutocompleteToolbarActivity.java

```diff
@@ -145,4 +190,45 @@ private void getPlacePredictions(String query) {
             }
         });
     }
+
+    /**
+     * Performs a Geocode API request and displays the result in a dialog
```

Comment:
Geocode API -> Geocoding API
Perhaps add a helpful link to https://developers.google.com/maps/documentation/geocoding/intro